src/Mod/Sketcher/Gui/TaskSketcherElements.cpp

```diff
@@ -1134,16 +1141,23 @@ void TaskSketcherElements::changeEvent(QEvent *e)
 void TaskSketcherElements::createSettingsButtonActions()
 {
     QAction* action = new QAction(QString::fromLatin1("Extended information"), this);
+    QAction* action2 = new QAction(QString::fromLatin1("Auto collapse filter"), this);

```

Comment:
This shall use `tr()` to be translatable (line above too).

Reply:
Like this ? 

```
    QAction* action = new QAction(tr("Extended information"), this);
    QAction* action2 = new QAction(tr("Auto collapse filter"), this);
```

Reply:
Yes